pkg/executor/insert_common.go

```diff
@@ -148,6 +149,14 @@ func (e *InsertValues) initInsertColumns() error {
 	} else {
 		// If e.Columns are empty, use all columns instead.
 		cols = tableCols
+		if e.Table.Meta().SoftdeleteInfo != nil ||
```

Comment:
If `tableInfo. SoftdeleteInfo != nil && !tableInfo. IsActiveActive`, we should also hide the column `_tidb_softdelete_time`.

The column `_tidb_commit_ts` shold be always be hidden by default.